source/editableText.py

```diff
@@ -42,21 +43,30 @@ class EditableText(ScriptableObject):
 	#: Whether or not to announce text found before the caret on a new line (e.g. auto numbering)
 	announceNewLineText=True
 
-	def _hasCaretMoved(self, bookmark, retryInterval=0.01, timeout=0.03):
+	def _hasCaretMoved(self, bookmark, retryInterval=0.01, timeout=None):

```

Comment:
In the description of this PR you mention backwards compat for this function I think? Changing timeout to `None` will risk breaking that.